File: vidralglass/src/app/model/endereco.ts
export class Endereco {
    idEndereco!: number;
    rua!: string;
    cidade!: string;
    bairro!: string;
    numero!: number;
    complemento!: string;
    idCliente!: number;

    constructor(){
        this.idEndereco = 0;
        this.rua = "";
        this.cidade = "";
        this.bairro = "";
        this.numero = 0;
        this.complemento = "";
        this.idCliente = 0;
    }
}

File: api/src/main/java/br/cefet/vidralglass/model/Endereco.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.cefet.vidralglass.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author criad
 */
@Data
@Getter
@Setter
@ToString
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class Endereco {
    protected int idEndereco;
    protected String rua;
    protected String cidade;
    protected String bairro;
    protected String numero;
    protected String complemento;
    protected int idCliente;
}
